lib/chef/resource/apt_package.rb

```diff
@@ -25,6 +25,35 @@ class AptPackage < Chef::Resource::Package
 
       provides :apt_package, target_mode: true
       provides :package, platform_family: "debian", target_mode: true
+      examples <<~DOC
+      **Install a package using package manager**:
+
+      ```ruby
+      apt_package 'name of package' do
+        action :install
+      end
+      ```
+
+      **Install a package without specifying the default action**:
+
+      ```ruby
+      apt_package 'name of package'
+      ```
+
+      **Install multiple packages at once**:
+
+      ```ruby
+      apt_package %(package1 package2 package3)
+      ```
+
+      **Install without using recommend packages as a dependency**
+
+      ```ruby
+      package 'apache2' do
```

Comment:
did you want this to be `apt_package` like the others?

Reply:
That was in the existing examples on the docs site and I think the intention was to show the `provides :package` for the debian platform_family